gabbi/tests/test_suite.py

```diff
@@ -54,3 +55,4 @@ def test_suite_catches_fixture_fail(self):
 
         self.assertIn('foo_alpha', str(errored_test))
         self.assertIn(VALUE_ERROR, trace)
+        self.assertIn(FIXTURE_METHOD, trace)
```

Comment:
👍

Reply:
I got the unicode missing rectangle symbol. I had to google it  :-)  ([👍 Pouce Vers Le Haut sur Apple iOS 11.1])